src/test/java/io/vertx/core/http/impl/EndPointKeyTest.java

```diff
@@ -24,7 +25,7 @@ public class EndPointKeyTest {
 
   @Test
   public void testEndPointKey() {
-    final SocketAddress addr = SocketAddress.inetSocketAddress(8080, "localhost");
+    final SocketAddress addr = SocketAddress.inetSocketAddress(HttpTestBase.DEFAULT_HTTP_PORT, "localhost");
```

Comment:
those tests don't use a server, so they should use literal values instead, that makes them less readable